cs/src/core/Index/Synchronization/HybridLogCheckpointTask.cs

```diff
@@ -12,13 +12,15 @@ namespace FASTER.core
     /// </summary>
     internal abstract class HybridLogCheckpointOrchestrationTask : ISynchronizationTask
     {
+        private long lastVersion;
```

Comment:
shouldnt this be initialized to -1 so that LastCheckpointedVersion will return -1 in case there are no checkpoints.

Reply:
I thought FASTER versions start at 1? So 0 is no checkpoint